erpnext_com/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md

```diff
@@ -15,8 +15,11 @@ When you create a new record you will have to specify
 1. The user for which the rule has to be applied
 2. The type of document which will be allowed (for example "Company")
 3. The specific item that you want to allow (the name of the "Company)
+4. Select "Is Default" to choose the default value that is allowed for a type of document.
```

Comment:
`If you check "Is Default", the default value will be set in the transaction for fields of this type (example "Company")`

Reply:
added suggestion